Formula/p/podman.rb

```diff
@@ -37,8 +37,10 @@ class Podman < Formula
     depends_on "fuse-overlayfs"
     depends_on "gpgme"
     depends_on "libseccomp"
+    depends_on "mpdecimal"
     depends_on "slirp4netns"
     depends_on "systemd"
+    depends_on "unzip"
```

Comment:
Were these deps added because of the previous job failures? They don't look related.

Reply:
Yes, CI was throwing errors that they were missing. We should remove them and rerun now.